gui.go

```diff
@@ -1391,7 +1391,7 @@ func (g *Gui) onKey(ev *GocuiEvent) error {
 			}
 		}
 
-		if ev.Key == MouseLeft && !v.Editable && g.openHyperlink != nil {
+		if ev.Key == MouseLeft && ev.Mod != ModMotion && !v.Editable && g.openHyperlink != nil {
```

Comment:
Shouldn't we say `(ev.Mod & ModMotion) != 0` instead? It's a bit mask.

(It's true that we don't include shift or alt for mouse events currently, but this might change in the future.)